packages/itwinui-react/src/core/Slider/Slider.test.tsx

```diff
@@ -196,7 +196,7 @@ it('should show tooltip when focused', async () => {
   expect(document.activeElement).toEqual(
     container.querySelector('.iui-slider-thumb'),
   );
-  expect(document.querySelector('.iui-tooltip')?.textContent).toBe('50');
+  expect(document.querySelector('.iui-tooltip')).toBeVisible();
```

Comment:
We can keep the `textContent` check too (similar to "should show tooltip on thumb hover" and "... focus" tests)

```suggestion
  const tooltip = document.querySelector('.iui-tooltip');
  expect(tooltip).toBeVisible();
  expect(tooltip?.textContent).toBe('50');
```

Reply:
the text is not important, and it's already checked in the other tests you mentioned. the goal here is to test visibility